guides/contributors/README.md

```diff
@@ -0,0 +1,99 @@
+# Contributors Guide
+
+## Index
+
+<ol>
```

Comment:
Could it be .md or .adoc is good but .html is tough to grok.

Reply:
Yeah, I've added HTML here because I can't do nested ordered lists in Markdown :/.

Reply:
Nevermind, I just realized you have to use 4-spaces indentation 🙄 :facepalm: